### PR TITLE
BL-2179: Do not select css files from the nonProductionStuff directory

### PR DESCRIPTION
**Pull request by phillip-hopper**

### CODE REVIEW

File: src/BloomExe/web/EnhancedImageServer.cs [hunk 1]
```diff
@@ -538,7 +538,8 @@ private bool ProcessCssFile(IRequestInfo info, string localPath)
 				if (_bloomBrowserUiCssFiles == null)
 				{
 					var sourceDir = FileLocator.GetDirectoryDistributedWithApplication("BloomBrowserUI");
-					_bloomBrowserUiCssFiles = Directory.EnumerateFiles(sourceDir, "*.css", SearchOption.AllDirectories).ToArray();
+					_bloomBrowserUiCssFiles = Directory.EnumerateFiles(sourceDir, "*.css", SearchOption.AllDirectories)
+						.Where(f => !f.Contains("nonProductionStuff")).ToArray();
```
Comment (JohnThomson) [inline src/BloomExe/web/EnhancedImageServer.cs]: Feels to me as if we're getting excessively complex here. First we try the OptimizedFileLocator, which is initialized with many of the directories in BloomBrowserUI. If they aren't found we then search ALL those directories, including the ones we already tried, in an arbitrary (alphabetical?) order. And now it turns out some of them have to be explicitly suppressed!
Could we not tell the file locator positively about ALL the directories that need to be tried, and get rid of this extra step altogether?
The one thing to be careful of would be whether some other code is using the file locator and would be made to look in additional, undesirable places. But I doubt we want different parts of Bloom using different search paths for CSS files. That's a recipe for confusion...another reason NOT to have a separate, additional mechanism here.

**Comment by hatton** [comment]
I assume the reason we don't just have the installer exclude this folder is that we don't want different behavior in production and dev environment? Next thing I wonder about is what this folder is for. Hmmm...

**Comment by JohnThomson** [comment]
The only reason I can think of for a folder with such a name is to be a place to temporarily put stuff we DO want to be different in production and dev...say, giving something an orange background that we want to quickly draw attention to if it shows up in an unexpected place. But if that's the purpose, of course, we should exclude it from the installer but NOT the paths the file locator searches. Alternatively, we could exclude such a folder from git so things in it never even get checked in.

**Comment by phillip-hopper** [comment]
Pull request updated

**Pull request merged**